dbt/include/synapse/macros/materializations/seed/seed.sql

```diff
@@ -45,6 +47,6 @@
     {{ return(statements[0]) }}
 {% endmacro %}
 
-{% macro sqlserver__load_csv_rows(model, agate_table) %}
-  {{ return(sqlserver__basic_load_csv_rows(model, 200, agate_table) )}}
+{% macro synapse__load_csv_rows(model, agate_table) %}
+  {{ return(synapse__basic_load_csv_rows(model, 200, agate_table) )}}
```

Comment:
@jtcohen6 found the solution to the error below. I had mistakenly deleted these lines assuming that `basic_load_csv_rows()` was dispatched. On to the next one!
> The SQL contains 0 parameter markers, but 30 parameters were supplied 'HY000